lib/room.rb

```diff
@@ -0,0 +1,30 @@
+require 'time'
+require_relative 'reservation'
+require_relative 'front_desk'
+
+
+  class Room
+
+    attr_reader :room_number, :room_reservations
```

Comment:
So rooms have reservations, but not blocks.  I would either pull reservations into your `FrontDesk` class or put blocks into room as well.